scripts/translation/libqa/OutputBuffer.php

```diff
@@ -0,0 +1,142 @@
+<?php /*
++----------------------------------------------------------------------+
+| Copyright (c) 1997-2025 The PHP Group                                |
++----------------------------------------------------------------------+
+| This source file is subject to version 3.01 of the PHP license,      |
+| that is bundled with this package in the file LICENSE, and is        |
+| available through the world-wide-web at the following url:           |
+| https://www.php.net/license/3_01.txt.                                |
+| If you did not receive a copy of the PHP license and are unable to   |
+| obtain it through the world-wide-web, please send a note to          |
+| license@php.net, so we can mail you a copy immediately.              |
++----------------------------------------------------------------------+
+| Authors:     André L F S Bacci <ae php.net>                          |
++----------------------------------------------------------------------+
+
+# Description
+
+This class caches formatted output, and calculates if this output is not
+previously marked as ignored, before printing it.                     */
+
+class OutputBuffer
+{
+    private string $filename = "";
+    private string $header = "";
+    private array  $matter = [];
+    private array  $footer = [];
+
+    private OutputIgnore $ignore;
+    private string $options;
+
+    function __construct( string $header , string $filename , OutputIgnore $ignore )
+    {
+        $filename = str_replace( "/./" , "/" , $filename );
+
+        $this->header = $header . ": " . $filename . "\n\n";
+        $this->filename = $filename;
+        $this->ignore = $ignore;
+
+        $copy = $ignore->residualArgv;
+        array_shift( $copy );
+        $this->options = implode( " " , $copy );
+    }
+
+    function add( string $text )
+    {
+        $this->matter[] = $text;
+    }
+
+    function addDiff( string $text , int $sourceCount , int $targetCount )
```

Comment:
I'm confused by the coding style, why not have `public` ? I know it is not required but I feel it would make sense to align with PER-CS.

Reply:
That is a simple change, I will do now.